cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -54,6 +54,8 @@ include:
     rules:
       - if: '$CI_COMMIT_TAG =~ /v[0-9]+\.[0-9]+\.[0-9]+$/'
         when: always
+      - if: '$CI_COMMIT_TAG =~ /GenomioDockerRebuild_v[0-9]+\.[0-9]+\.[0-9]+[a-z]$/'
```

Comment:
do you need versions in the regexp at all?

Reply:
Yes Imho we need it, as it will be added to the docker tag as well as for tracking manual tags made with git tag. Also its needed to understand which genomio python version the container  actually is.